packages/react-components/react-tabs/library/src/components/Tab/useTab.ts

```diff
@@ -79,7 +82,7 @@ export const useTab_unstable = (props: TabProps, ref: React.Ref<HTMLElement>): T
     icon: iconSlot,
     iconOnly,
     content: contentSlot,
-    contentReservedSpace: slot.optional(content, {
+    contentReservedSpace: slot.optional(contentReservedSpace, {
```

Comment:
_5 cents_:
I am not a big fun of this design: it should be two separate slots (`content` + `contentReservedSpace`) (if it's really needed), so they could be targeted separately. Anyway, it's too late to complain about it.